open_seq2seq/data/speech2text/speech_utils.py

```diff
@@ -317,7 +385,9 @@ def get_speech_features_librosa(signal, sample_freq, num_features,
                                 num_fft=None,
                                 dither=0.0,
                                 norm_per_feature=False,
-                                mel_basis=None):
+                                mel_basis=None,
+                                custom_noise=None,
+                                prob_noise=[0.5, 0.5, 0.5]):
```

Comment:
These default values for probs thresholds break backward compatibility with the previous version of `master` branch.